backend/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:alpine AS builder
+
```

Comment:
Can you add some comments here please ?
in the future this file can became bigger

Reply:
Sure let me do that 👍